applications/asset_tracker_v2/src/modules/Kconfig.app_module

```diff
@@ -10,4 +10,7 @@ config APP_REQUEST_GPS_ON_INITIAL_SAMPLING
 	bool "Request GPS on initial data sampling after connection"
 	default y if GPS_MODULE
 
+config APP_REQUEST_NEIGHBOR_CELLS_DATA
+	bool "Include neighbor cells data for regular sample requests"
```

Comment:
This could use a help text to be used in doc generation and a bit less wordy description.
Maybe something like this:
```
config APP_REQUEST_NEIGHBOR_CELLS_DATA
	bool "Include neighbor cells data"
        help
          Include LTE neighbor cell measurement data in regular sampling requests.
          The data will be encoded together with the other sampled data and sent to cloud, where it can be used to determine the device's location.
```

Reply:
Nice suggestion. I'll add it.